Rakefile

```diff
@@ -103,3 +103,8 @@ task(:commits) do
     puts "...passed"
   end
 end
+
+# Add i18n tasks (gettext:pot, gettext:po*)
```

Comment:
Do I don't know that much about rake: when does this run? (i.e. when does the pot file get generated?)